src/credentials.ts

```diff
@@ -162,7 +162,7 @@ export class CredentialKeeper
   async getCredentialType(cred: SignedCredential): Promise<CredentialType> {
     const metadata = await this.storage.get.credentialMetadata(cred)
 
-    if (!metadata.issuer) {
+    if (metadata.issuer) {
```

Comment:
This was "!metadata.issuer" because sometimes the metadata was actually incomplete. I will try to add a test case for that.

I think the right way to handle this is to first take the issuer DID from either one of `cred.issuer` or `metadata.issuer`, then look up the public profile